egs/chime4/ConvTasNet/run.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+set -o pipefail
+
+# The root directory containing CHiME3
+storage_dir=
+
+# Directory containing the pretrained model
+exp_dir=
+# After running the recipe a first time, you can run it from stage 3 directly to train new models.
+
+# Path to the python you'll use for the experiment. Defaults to the current python
+# You can run ./utils/prepare_python_env.sh to create a suitable python environment, paste the output here.
+python_path=python
+
+# Example usage
+# ./run.sh --stage 3 --tag my_tag --task sep_noisy --id 0,1
+
+# General
+stage=0  # Controls from which stage to start
+tag=""  # Controls the directory name associated to the experiment
+# You can ask for several GPUs using id (passed to CUDA_VISIBLE_DEVICES)
+
+eval_use_gpu=0
+# Need to --compute_wer 1 --eval_mode max to be sure the user knows all the metrics
+# are for the all mode.
+compute_wer=1
+
+# Choice for the ASR model whether trained on clean or noisy data. One of clean or noisy
+asr_type=noisy
+
+test_dir=data/test
```

Comment:
Does this need to be overwritten? 
If it doesn't, then add it after utils/parse_options.sh